src/client/pages/Offer/utils.ts

```diff
@@ -453,6 +455,15 @@ export const getBundleVariantFromInsuranceTypesWithFallback = (
 
   if (matchingVariant) return matchingVariant
 
+  const matchingPreviousInsurance = variants.find(
```

Comment:
Same logic as in InsuranceSelector, does it make sense to extract it?

Reply:
Let me address that in a follow-up PR since we also need to match on exposure :)